modules/navigation/nav_utils.h

```diff
@@ -298,6 +298,19 @@ class Heap {
 		}
 	}
 };
+
+struct PerformanceData {
+	int pm_region_count = 0;
```

Comment:
If the struct is called "performance_data" why are all members still prefixed with `pm`, which I assume means "performance"?

Reply:
I had the prefix originally removed but reverted and added it back in. Without that prefix it turns the code readability into a shitshow when you dont have a more sophisticated IDE as your mental crutch available. There are so many similar temp variables all over the place in sub functions that can be easily confused and the pm_ prefix makes it very clear what is what.

Reply:
I agree with @Mickeon - as the fields are always prefixed with `[name].` it's possible to have good readability without that `pm_` prefix